### PR TITLE
refs platform/2294: Add Content Security Policy CSP Headers

### PR DESCRIPTION
**Pull request by filippolmt**

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -40,6 +40,15 @@ if [ "${NGINX_HSTS_MAX_AGE}" -gt 0 ]; then
   sed -e '/#hstsheader/r /templates/hsts.conf' -i /templates/from-to-www.conf.tpl;
 fi
 
+# Activate CSP header (default: off)
+# https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
+export NGINX_CSP_HEADER="${NGINX_CSP_HEADER}"
```
Comment (Monska85) [inline docker-entrypoint.sh]:
```suggestion
export NGINX_CSP_HEADER="${NGINX_CSP_HEADER:-}"
```

**Review by Monska85 — CHANGES_REQUESTED** [review]
Please add test to check that the header is not present if the variable is empty

**Review by Monska85 — APPROVED** [review]
LGTM :+1:
